common/shared.c

```diff
@@ -747,6 +747,18 @@ int ft_get_cq_fd(struct fid_cq *cq, int *fd)
 	return ret;
 }
 
+int ft_startup(void)
+{
+	int ret = 0;
+#if defined(_WIN32)
+	WSADATA data;
+	ret = WSAStartup(MAKEWORD(2, 2), &data);
+	if (ret)
+		return HRESULT_FROM_WIN32(ret);
+#endif
+	return ret;
+}
```

Comment:
Please move this into the osd.h (or osd.c) files for the respective builds.  We should strive to avoid having any ifdef's in any common C code.